src/FormMigration/Steps/MailchimpSettings.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class MailchimpSettings extends FormMigrationStep
+{
+    /**
+     * @unreleased
+     */
+    public function process(): void
+    {
+        $prevFormId = $this->formV2->id;
+
+        $isFormEnabled = $this->getFormMetaValue($prevFormId, '_give_mailchimp_enable') === 'true';
```

Comment:
I would suggest adding these calls to the `FormMetaDecorator`, which is a more declarative approach.

For example, you can call `$this->formV2->getDonationGoalAmount()`, which encapsulates where/how that value is stored.

In this case, it could be `$this->formV2->isMailchimpEnabled()`.

Also, instead of directly comparing against `true`, consider the function helper `give_is_setting_enabled()`.

Reply:
Ok, got it. Thanks! a0161125118cc5af042dc2b0087be48f697359db